app/hooks/useLiquidityPosition.ts

```diff
@@ -394,14 +405,10 @@ export function useLiquidityPosition({
           });
         }
 
-        // Handle EVM chain withdrawals
-        const { depositWithExpiry } = useContracts({
-          tokenAddress: tokenAddress as Address | undefined,
-          provider: wallet?.provider,
-          assetId: pool.asset,
-        });
         await switchEvmChain(wallet.provider, assetChain);
 
+        await delay(5000);
```

Comment:
why do we add a delay here? if it's need, can we please add a comment on top of it?